integration-tests/google-pubsub/pom.xml

```diff
@@ -39,7 +39,18 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-mock</artifactId>
```

Comment:
Add the additional CQ dependencies to the `virtualDependencies` profile.

Reply:
ok, thanks